go.mod

```diff
@@ -136,17 +139,21 @@ require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.4 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dimchansky/utfbom v1.1.1 // indirect
+	github.com/docker/distribution v2.8.3+incompatible // indirect
 	github.com/docker/docker-credential-helpers v0.8.2 // indirect
 	github.com/docker/go-events v0.0.0-20190806004212-e31b211e4f1c // indirect
 	github.com/docker/go-metrics v0.0.1 // indirect
+	github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7 // indirect
```

Comment:
😬 That's what I was fearing would be the case; the `registry` package in the CLI depends on the deprecated registry client, which depends on the registry API (server!) code, which depends on `gorilla/mux` as well as the very deprecated `libtrust` package 😞 

- related https://github.com/moby/moby/issues/46940